src/pudl/scripts/dbt_helper.py

```diff
@@ -616,8 +595,8 @@ def validate(
 
     build_params = {
         "node_selection": node_selection,
-        "dbt_target": target,
         "node_exclusion": exclude,
+        "dbt_target": "etl-full",
```

Comment:
This is a required argument -- the script crashes without it. I think we want it hard coded to be `etl-full` -- but if there's ever anything else that varies by target other than the row counts, we'll need to remember this.